src/main/java/dan200/computercraft/core/apis/TermAPI.java

```diff
@@ -67,13 +73,17 @@ public void shutdown( )
             "getTextColor",
             "getBackgroundColour",
             "getBackgroundColor",
-            "blit"
+            "blit",
+            "setColour",
```

Comment:
I've had a rethink of these method names. I think to avoid solution with text/background colour we should rename them setPaletteColo(u)r